multigeiger/thp_sensor.cpp

```diff
@@ -4,29 +4,61 @@
 
 #include <Adafruit_Sensor.h>
 #include <Adafruit_BME280.h>
+#include <Adafruit_BME680.h>
 
 #include "log.h"
 #include "thp_sensor.h"
 
-static bool have_thp = false;
+static bool have_thp;
+static int type_thp = 0;
 
-Adafruit_BME280 bme;
+Adafruit_BME280 bme280;
+Adafruit_BME680 bme680;
 
 bool setup_thp_sensor(void) {
```

Comment:
this could return the type of the detected sensor or 0 for none.

Reply:
Okay, will simplify